### DISCUSSION TITLE
Neovintageousrc Location

### DISCUSSION THREAD

**Discussion by kvnduff** [gerardroche/gerardroche.github.io#52] [General]
Hi Gerard, I'm new to Sublime and Neovintageous... just starting out. Looking to create some custom keymappings in the rc file. One question I couldn't find in the forums or help docs... what folder should the neovintageous file be saved to using Windows 10? Sorry for the basic question and thanks for any guidance you can provide.

**Comment by gerardroche** [comment]
Use the Command Palette: `NeoVintageous: Open RC File`.

Don't forget to reload it, use the Command Palette: `NeoVintageous: Reload RC File`

It's inside the sublime packages user directory: `path/to/sublime/Packages/User/.neovintageousrc`

Find it via **Menu > Browse Packages**.

Neo docs here: https://github.com/NeoVintageous/NeoVintageous/blob/master/res/doc/neovintageous.txt

And you can run `:help nv` to access them from within sublime.

**Comment by kvnduff** [comment]
Hi Gerard, thanks so much for your response! When I use the command "NeoVintageous: Open RC File", I always receive an invalid command error. Do you know why this might be?

**Comment by gerardroche** [comment]
Hmm, that sounds like a bug but I can't reproduce it.

Can you show some console debug information?

> Show the Sublime Text console log: Menu > View > Show Console.
>
> Command and input logging are enabled by running the following commands in input box at the bottom of the console: sublime.log_commands(True) and sublime.log_input(True).

https://github.com/NeoVintageous/NeoVintageous/blob/master/.github/CONTRIBUTING.md

**Comment by kvnduff** [comment]
Hi Gerard, I'm really sorry that I forgot to get back to you on this. This was really an oversight (a stupid one) on my part. I was attempting to type "NeoVintageous: Reload RC File" on the nv ex command line and I obviously should have been seeking the command from the Sublime command palette. Really sorry about that. I'll close this off.

**Comment by gerardroche** [comment]
No worries. Glad the issue is resolved.
